staging/traefik/templates/deployment.yaml

```diff
@@ -161,30 +175,41 @@ spec:
           containerPort: 8880
           protocol: TCP
         - name: https
-          containerPort: 443
+          containerPort: {{ template "traefik.containerPort.https" . }}
           {{- if .Values.deployment.hostPort.httpsEnabled }}
           hostPort: {{ default 443 .Values.deployment.hostPort.httpsPort }}
           {{- end }}
           protocol: TCP
-        {{- if .Values.dashboard.enabled }}
+        {{- if or .Values.dashboard.enabled .Values.metrics.prometheus.enabled }}
```

Comment:
hum, this looks weird. is that what upstream is doing as well?

Reply:
yeah, this is pulled from upstream, looks like it was intro'd [here](https://github.com/helm/charts/pull/15210). doesn't look like that change is needed now that dashboard and prometheus metrics are decoupled, maybe they missed this line. I can revert the line and also open a pr against upstream

Reply:
https://github.com/helm/charts/pull/23156